### PR TITLE
Support Java examples

### PR DESCRIPTION
**Pull request by darionyaphet**

### CODE REVIEW

File: client/src/main/java/com/vesoft/nebula/client/example/GraphClientExample.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            if (code != ErrorCode.SUCCEEDED) {
+                LOGGER.error(String.format("Insert Edges Failed: %s", statement));
+                System.exit(-1);
+            }
+        }
+
+        ResultSet resultSet = client.executeQuery("GO FROM 201 OVER like;");
+        List<String> columns = resultSet.getColumns()
+                                        .stream()
+                                        .map(bytes -> new String(bytes))
```
Comment (yixinglu) [inline client/src/main/java/com/vesoft/nebula/client/example/GraphClientExample.java]: Can simplify this map:
```
.map(String::new)
```

File: client/src/main/java/com/vesoft/nebula/client/example/GraphClientExample.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            LOGGER.info("ID: %d", value.columns.get(0).getId());
+        }
+
+        resultSet = client.executeQuery("GO FROM 201 OVER like " +
+                "WHERE $$.student.age >= 17 YIELD $$.student.name AS Friend, " +
+                "$$.student.age AS Age, $$.student.gender AS Gender;");
+
+        columns = resultSet.getColumns()
+                .stream()
+                .map(bytes -> new String(bytes))
```
Comment (yixinglu) [inline client/src/main/java/com/vesoft/nebula/client/example/GraphClientExample.java]: ditto

File: client/src/main/java/com/vesoft/nebula/client/example/GraphClientExample.java [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+                                        .collect(Collectors.toList());
+        LOGGER.info(String.format("Columns: %s", Joiner.on(" ").join(columns)));
+
+        for (RowValue value : resultSet.getRows()) {
+            LOGGER.info("ID: %d", value.columns.get(0).getId());
+        }
+
+        resultSet = client.executeQuery("GO FROM 201 OVER like " +
+                "WHERE $$.student.age >= 17 YIELD $$.student.name AS Friend, " +
+                "$$.student.age AS Age, $$.student.gender AS Gender;");
```
Comment (yixinglu) [inline client/src/main/java/com/vesoft/nebula/client/example/GraphClientExample.java]: try... catch `executeQuery` exception

File: client/src/main/java/com/vesoft/nebula/client/example/GraphClientExample.java [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+        LOGGER.info(String.format("Columns: %s", Joiner.on(" ").join(columns)));
+
+        for (RowValue value : resultSet.getRows()) {
+            LOGGER.info("%s, %d, %s", new String(value.columns.get(0).getStr()),
+                                      value.columns.get(1).getInteger(),
+                                      new String(value.columns.get(2).getStr()));
+        }
+
+        client.disconnect();
```
Comment (yixinglu) [inline client/src/main/java/com/vesoft/nebula/client/example/GraphClientExample.java]: try... catch ... finally { client.disconnect(); }

File: client/src/main/java/com/vesoft/nebula/client/example/GraphClientExample.java [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+            "INSERT EDGE select(grade) VALUES 200 -> 101:(5);",
+            "INSERT EDGE select(grade) VALUES 200 -> 102:(3);",
+            "INSERT EDGE select(grade) VALUES 201 -> 102:(3);",
+            "INSERT EDGE select(grade) VALUES 202 -> 102:(3);",
+            "INSERT EDGE like(likeness) VALUES 200 -> 201:(92.5);",
+            "INSERT EDGE like(likeness) VALUES 201 -> 200:(85.6);",
+            "INSERT EDGE like(likeness) VALUES 201 -> 202:(93.2);"
+    };
+
+    public static void main(String[] args) throws ConnectionException, TException, NGQLException {
```
Comment (yixinglu) [inline client/src/main/java/com/vesoft/nebula/client/example/GraphClientExample.java]: Shall we throw exceptions in `main` function ?

Comment (darionyaphet) [reply client/src/main/java/com/vesoft/nebula/client/example/GraphClientExample.java]: Good point Actually I'm not sure but catch exception seems better :)

File: client/src/main/java/com/vesoft/nebula/client/example/GraphClientExample.java (hunk 3 above)
Comment (darionyaphet) [reply client/src/main/java/com/vesoft/nebula/client/example/GraphClientExample.java]: `GraphClient` extends `AutoCloseable` and implement close()

File: client/src/main/java/com/vesoft/nebula/client/example/GraphClientExample.java (hunk 4 above)
Comment (darionyaphet) [reply client/src/main/java/com/vesoft/nebula/client/example/GraphClientExample.java]: GraphClient extends AutoCloseable and implement close()

File: client/src/main/java/com/vesoft/nebula/client/example/GraphClientExample.java (hunk 1 above)
Comment (darionyaphet) [reply client/src/main/java/com/vesoft/nebula/client/example/GraphClientExample.java]: I like it  really good idea 👍

**Pull request merged**